src/main/scala/se/nimsa/dicom/data/VR.scala

```diff
@@ -40,5 +40,5 @@ object VR extends Enumeration {
   final val UT = Val(0x5554, 12, ' ')
 
   private lazy final val valueOfMap: Map[Int, Value] = VR.values.map(vr => vr.code -> vr).toMap
-  def valueOf(code: Int): VR = try valueOfMap(code) catch { case _: Throwable => VR.UN }
+  def valueOf(code: Int): VR = try valueOfMap(code) catch { case _: Throwable => null }
```

Comment:
Make it possible to differ between when an element actually has VR UN and when VR could not be parsed